tslint.json

```diff
@@ -6,14 +6,15 @@
     "trailing-comma": [false],
     "object-literal-sort-keys": false,
     "arrow-parens": false,
-    "quotemark": ["single"],
+    "quotemark": [true, "single"],
```

Comment:
fixed to have the correct type according to https://palantir.github.io/tslint/rules/quotemark/